scripts/linters/pylint_extensions_test.py

```diff
@@ -2234,3 +2234,191 @@ def test_file_overview_at_end_of_file(self):
 
         with self.checker_test_object.assertAddsMessages(message):
             temp_file.close()
+
+
+class NewlineBelowClassDocstringTests(unittest.TestCase):
+
+    def setUp(self):
+        super(NewlineBelowClassDocstringTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.NewlineBelowClassDocstring)
+        self.checker_test_object.setup_method()
+
+    def test_no_newline_below_class_docstring(self):
+        node_no_newline_below_class_docstring = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(Class):
```

Comment:
why the param is `Class`? (Ditto elsewhere)

Reply:
This is just a dummy class so I used class here. Just random.

Reply:
changed it to object.